guides/common/modules/proc_configuring-satellite-to-use-external-databases.adoc

```diff
@@ -28,3 +28,19 @@ Use the `{foreman-installer}` command to configure {Project} to connect to an ex
   --foreman-proxy-content-pulpcore-postgresql-db-name pulpcore \
   --foreman-proxy-content-pulpcore-postgresql-password _Pulpcore_Password_
 ----
++
+[NOTE]
+====
```

Comment:
Can we not just put it as running text? 
If we were to add a note every time, we would have nothing but notes. 
Everyone wants a note, but not everything is noteworthy. This just looks like information.

Reply:
@melcorr Thank you for the review! Sure, you're right :-) I'll fix it in the next commit. Thank you!